### PR TITLE
Trev8939/feedback fixes

### PR DESCRIPTION
**Pull request by TADraeseke**
@LDuncAndroid - can you review this when you have a moment. Small changes to 4 samples based on feedback:

- Disable clip button after first clip (in Clip Geometry)
- Add a side scroll bar and dividing lines to recycler view (Search for Webmap)
- Set anchor point to center and add explanatory comment (Scene Property Expression)
- Remove borders on feature service area result geometries

### CODE REVIEW

File: java/clip-geometry/src/main/java/com/esri/arcgisruntime/sample/clipgeometry/MainActivity.java [hunk 1]
```diff
@@ -101,6 +101,8 @@ protected void onCreate(Bundle savedInstanceState) {
           clipAreasOverlay.getGraphics().add(clippedGraphic);
         }
       }
+      // disable button
+      clipButton.setEnabled(false);
```
Comment (LDuncAndroid) [inline java/clip-geometry/src/main/java/com/esri/arcgisruntime/sample/clipgeometry/MainActivity.java]: Should you disable the button directly before performing the clip operation? The clip operations may not be instantaneous leaving a chance to click it again.

File: java/find-service-area-interactive/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.esri.arcgisruntime.sample.findservicearea">
+    package="com.esri.arcgisruntime.sample.findserviceareainteractive">
 
-    <uses-permission android:name="android.permission.INTERNET"/>
+    <uses-permission android:name="android.permission.INTERNET" />
```
Comment (LDuncAndroid) [inline java/find-service-area-interactive/src/main/AndroidManifest.xml]: This is missing `uses-feature` for OpenGL.

File: java/find-service-area-interactive/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -11,7 +11,9 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
-        <activity android:name=".MainActivity">
+        <activity
+            android:name="com.esri.arcgisruntime.sample.findserviceareainteractive.MainActivity"
```
Comment (LDuncAndroid) [inline java/find-service-area-interactive/src/main/AndroidManifest.xml]: If the directory structure matches the package name, this should still be `.MainActivity`

File: java/search-for-webmap/src/main/AndroidManifest.xml [hunk 4]
```diff
@@ -2,7 +2,7 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.esri.arcgisruntime.sample.searchforwebmap">
 
-    <uses-permission android:name="android.permission.INTERNET"/>
+    <uses-permission android:name="android.permission.INTERNET" />
```
Comment (LDuncAndroid) [inline java/search-for-webmap/src/main/AndroidManifest.xml]: Missing `uses-feature` for OpenGL.

File: java/search-for-webmap/src/main/res/layout/activity_main.xml [hunk 5]
```diff
@@ -18,14 +18,15 @@
...
+        android:text="@string/to_search_text" />
 
     <android.support.v7.widget.RecyclerView
         android:id="@+id/webmapRecyclerView"
         android:layout_width="240dp"
         android:layout_height="match_parent"
         android:layout_gravity="start"
-        android:background="@color/drawerBackground"/>
+        android:background="@color/drawerBackground"
+        android:scrollbars="vertical" />
```
Comment (LDuncAndroid) [inline java/search-for-webmap/src/main/res/layout/activity_main.xml]: Is this necessary?

Comment (TADraeseke) [reply java/search-for-webmap/src/main/res/layout/activity_main.xml]: It was one of the feedback suggestions for the samples team to give an indication of where on the list the user is.

Comment (LDuncAndroid) [reply java/search-for-webmap/src/main/res/layout/activity_main.xml]: Sorry, missed that as a requirement.

**Comment by TADraeseke** [comment]
Thanks @LDuncAndroid - Made those changes. The scrollbar was requested by members of the sample team to make the list easier to navigate.

**Pull request merged**